### ISSUE TITLE
About the 71.4 NDS in Nus testset config

### ISSUE THREAD

**Issue by StiphyJay** [agent-sgs/PillarNet#2]
Hi, Thanks for your surperising work.

I wanted to reproduce the performance of 71.4 NDS you implemented on the Nuscenes test 、set, but when I trained on the Nus dataset with configs/nusc/voxelnet/nusc_centerpoint_pilarnet_h_iouoDiou.py, I only got the following performance：
![image](https://user-images.githubusercontent.com/28260101/174435215-8f6ea0e0-5075-4be6-9a74-e38c99bbd2a9.png)

 Could you provide the corresponding config file about the highest precision reported in your paper?

Looking forward to your reply!

Best!

**Comment by agent-sgs** [comment]
I will provide the details for the performance on the nuScenes test set.

**Comment by StiphyJay** [comment]
> I will provide the details for the performance on the nuScenes test set.

> I will provide the details for the performance on the nuScenes test set.

Thanks for your reply. Another problem is that we found in the training process of the network, when we use fade_epochs , the network will get stuck here after using fade strategy in the new training epoch, and there will be no further training, and there will be no error. Have you ever encountered such a problem?  How to fix it?
![image](https://user-images.githubusercontent.com/28260101/176131013-509eb818-ee36-4f08-91f0-ea799d779e9f.png)

**Comment by AndyYuan96** [comment]
> > I will provide the details for the performance on the nuScenes test set.
> 
> > I will provide the details for the performance on the nuScenes test set.
> 
> Thanks for your reply. Another problem is that we found in the training process of the network, when we use fade_epochs , the network will get stuck here after using fade strategy in the new training epoch, and there will be no further training, and there will be no error. Have you ever encountered such a problem? How to fix it? ![image](https://user-images.githubusercontent.com/28260101/176131013-509eb818-ee36-4f08-91f0-ea799d779e9f.png)

Hi,can you reproduce the test set's result.

**Comment by StiphyJay** [comment]
> 
Yes. I reproduce the paper result (pillarnet-res18), NDS-70.8, map-65.0 on nuScenese test set, with using fade strategy and TTA-flip.

**Comment by study1994** [comment]
> Yes. I reproduce the paper result (pillarnet-res18), NDS-70.8, map-65.0 on nuScenese test set, with using fade strategy and TTA-flip.

@StiphyJay reproduce the paper result from map-0.577 to map-65.0 just using fade strategy and TTA-flip? did you change other code?
